core/src/snapshot_packager_service.rs

```diff
@@ -394,42 +392,56 @@ mod tests {
         fs::create_dir_all(&incremental_snapshot_archives_dir).unwrap();
 
         fs::create_dir_all(&accounts_dir).unwrap();
+
+        let slot: Slot = 4;
+
         // Create some storage entries
         let storage_entries: Vec<_> = (0..5)
             .map(|i| Arc::new(AccountStorageEntry::new(&accounts_dir, 0, i, 10)))
             .collect();
 
-        // Create some fake snapshot
-        let snapshots_paths: Vec<_> = (0..5)
-            .map(|i| {
-                let snapshot_file_name = format!("{i}");
-                let snapshots_dir = snapshots_dir.join(&snapshot_file_name);
-                fs::create_dir_all(&snapshots_dir).unwrap();
-                let fake_snapshot_path = snapshots_dir.join(&snapshot_file_name);
-                let mut fake_snapshot_file = OpenOptions::new()
-                    .read(true)
-                    .write(true)
-                    .create(true)
-                    .open(&fake_snapshot_path)
-                    .unwrap();
-
-                fake_snapshot_file.write_all(b"Hello, world!").unwrap();
-                fake_snapshot_path
-            })
-            .collect();
+        // Create one fake snapshot.
+        let snapshot_file_name = format!("{slot}");
+        let snapshot_dir = snapshots_dir.join(&snapshot_file_name);
+        fs::create_dir_all(&snapshot_dir).unwrap();
+        let fake_snapshot_path = snapshot_dir.join(&snapshot_file_name);
+        let mut fake_snapshot_file = OpenOptions::new()
+            .read(true)
+            .write(true)
+            .create(true)
+            .open(&fake_snapshot_path)
+            .unwrap();
+        fake_snapshot_file.write_all(b"Hello, world!").unwrap();
+        let fake_status_cache = snapshot_dir.join(SNAPSHOT_STATUS_CACHE_FILENAME);
+        let mut fake_status_cache_file = OpenOptions::new()
+            .read(true)
+            .write(true)
+            .create(true)
+            .open(fake_status_cache)
+            .unwrap();
+        fake_status_cache_file
+            .write_all(b"Hello, status cache!")
+            .unwrap();
```

Comment:
I realize it was using `OpenOptions` before, but this seems equivalent and a lot less cluttered.
```rust
        let mut fake_snapshot_file = File::create(&fake_snapshot_path).unwrap();
        fake_snapshot_file.write_all(b"Hello, world!").unwrap();

        let fake_status_cache = snapshot_dir.join(SNAPSHOT_STATUS_CACHE_FILENAME);
        let mut fake_status_cache_file = File::create(&fake_status_cache).unwrap();
```

Reply:
Unless I am missing something that makes this different.

Reply:
Not in this PR any more.